.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature Request] Title"
+labels: ''
+assignees: ''
+
+---
+
+## Is your feature request related to an existing problem?  Please describe.
+
+`[Author TODO: A clear and concise description of what the problem is.]`
+
+## Describe the desired solution
+
+`[Author TODO: A clear and concise description of how you would like the feature to work.]`
+
+## Describe possible alternatives
```

Comment:
Sometimes a feature would simplify the stack like for example macvtap vs tap. It would make sense that this paragraph refers also to how one works around this gap in functionality.

Reply:
I *think* I got your point,  but happy to change if not...